src/main/java/com/bina/lrsim/H5Sampler.java

```diff
@@ -87,6 +87,12 @@ else if(this.reference.exists()) {
         } else {
           log.error("ccs spec is supported only with cmp.h5 input, which contains beyond-fastq quality scores, please contact developer if alternatives are needed");
         }
+      } else if (readType.equals("clrbam")) {
+        if (this.inFile.endsWith("bam")) {
```

Comment:
Might be better to check for ending with ".bam" since that is the proper extension.